Modules/PSGraph.md

```diff
@@ -0,0 +1,21 @@
+# Name: PSGraph
+
+## Description:
+PSGraph is a module that is implemented as a DSL for the GraphViz engine. This allows you to generate graphs of information with Nodes and Edges dynamically from your data.
+
+| Works with | Author(s) | Project URL(s) | Creation Date | Tags |
+|------------|--------|-------------------|---------------|------|
+| [GraphViz](http://graphviz.org/) | Kevin Marquette (<a href="https://kevinmarquette.github.io/blog/?utm_source=PowerShellModulesCentral&utm_medium=markdown" target="_blank">b</a> \| <a href="https://twitter.com/kmarquette" target="_blank">t</a>) | [Github](https://github.com/KevinMarquette/PSGraph) \| [official project site](https://kevinmarquette.github.io/) | Jan 2017 | Graph, DSL, Visualization |
```

Comment:
It seems that your twitter handle is wrong. 😄 
should be this one - https://twitter.com/KevinMarquette - right?

Reply:
yes, thank you.